website/docusaurus.config.js

```diff
@@ -215,7 +215,7 @@ const config = {
           postsPerPage: 'ALL',
           feedOptions: {
             type: 'all',
-            copyright: `Copyright © ${new Date().getFullYear()} Podman Desktop`,
+            copyright: `Copyright © ${new Date().getFullYear()} Red Hat, Inc.`,
```

Comment:
I would not mix the change as it's not related. I think it can kept like this

Podman.io has 'Copyright © 2023 Podman. Built with Docusaurus.'

Reply:
That's confusing.

Reply:
This is related: to add the licence properly, the copyright must be properly attributed.

1. This copyright can be mistaken for the copyright on the application.
2. "Podman Desktop" is not a legal entity.